### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by 35612angrymen** [gucci-on-fleek/lockdown-browser#34] [Q&A] [answered]
As long as we wait for the browser to open by itself and don't double click the icon and if we are able to login and navigate freely it should be undetected?

**Comment by gucci-on-fleek** [comment, marked as answer]
As far as I'm aware, as long as the _Browser_ opens at all you should be fine. I've never seen any evidence that it will detect that something is fishy, let you log in, then report it to your school.

Of course, an update could change this at any time, so _caveat emptor_.
